cpp/include/experimental/graph_view.hpp

```diff
@@ -114,6 +114,18 @@ class partition_t {
     }
   }
 
+  int get_row_size() const {
+    return row_comm_size_;
+  }
+
+  int get_col_size() const {
+    return col_comm_size_;
+  }
+
+  int get_comm_rank() const {
+    return comm_rank_;
+  }
+
```

Comment:
Better delete this. Retrieve row_comm_size_, col_comm_size_, an comm_rank_ from handle; better keep row_comm_size_, col_comm_size_, comm_rank_ as internal to partition_t as originally intended.

Reply:
I'm inclined to address this in 0.17.

There's a bigger design issue to resolve related to graph contraction.  The graph contraction work-around that's implemented in this PR essentially replicates the data in the partition_t object with a different interface to support shuffling data around the GPU cluster.  I could adjust these and return them to internal, but I'll just have to revisit all of this again in 0.17 anyway.

Admittedly, I use these for multiple purposes in this PR, not just creating the compute_partition_t object.  But once the compute_partition_t object issue is refactored we can address this more easily.

Reply:
Sure, once you're back, you may submit a PR just saying this is temporary and to be removed; basically discouraging others in the cuGraph team from using this.